planet/cli/data.py

```diff
@@ -135,6 +135,22 @@ async def search_create(ctx, name, item_types, filter, daily_email, pretty):
         echo_json(items, pretty)
 
 
+@data.command()
+@click.pass_context
+@translate_exceptions
+@coro
+@pretty
+@click.argument('search_id')
+async def search_get(ctx, search_id, pretty):
+    """ This function obtsains an existing saved search, using the search_id.
+    This function outputs a full JSON description of the identified saved
+    search. The output can also be optionally pretty-printed using "--pretty".
+    """
```

Comment:
I'd recommend updating this to have the first line be a short description and the longer description to follow. `planet orders get` as reference: https://github.com/planetlabs/planet-client-python/blob/a65a117ebba604a9c0e0c5dd2a95b5e519255ad3/planet/cli/orders.py#L84